src/interfaces/standings.interface.ts

```diff
@@ -0,0 +1,51 @@
+  import {Problem} from './problem.interface'
+  import {Member} from './member.interface'
+  
+
+  export interface Standings {
+    contest: Contest
+    problems: Problem[]
+    rows: Row[]
+  }
+  
+  export interface Contest {
```

Comment:
1. I would suggest moving this Contest interface to its own separate file.
2. Please complete the type according to the API docs: https://codeforces.com/apiHelp/objects#Contest